lib/ValidateDirectiveVisitor.test.ts

```diff
@@ -1382,8 +1382,20 @@ directive @${name}(
                 arg: [DeepNullableEntry]
                 validationErrors: [ValidatedInputErrorOutput!]
               }
+              input YetAnotherInput {
+                number: Int
+              }
+              input Args {
+                field: String!
+                moreInputs: YetAnotherInput
+              }
+              input NullableSubInput {
+                field1: Args
+                field2: Args
+              }
               input DeepNullableInput {
                 nullable: Int @${name}
+                nullable2: NullableSubInput
```

Comment:
I don't get why do you need these declarations to test, none are using the directive...